validmind-docs.yaml

```diff
@@ -0,0 +1,76 @@
+---
```

Comment:
@nibalizer as mentioned in the root comment of the PR, I'd love your help with verifying that this behaves as expected. I have tinkered a bit with `kubectl apply -f validmind-docs.yaml` but nothing like what a real-life environment might require. I'd like to make sure this passes muster with you before we ship.

Reply:
yep this seems fine.